docusaurus/video/docusaurus/docs/api/basics/authentication.mdx

```diff
@@ -244,6 +349,22 @@ def create_user_token(request):
 
 ```
 
+</TabItem>
+
+<TabItem value="bash" label="Bash">
+
+```bash
```

Comment:
I tested, and the resulting token is accepted by the Stream API, but please still review just in case I missed something